examples/dependencies/app/components/my-chart.js

```diff
@@ -1,12 +1,14 @@
-import Component from '@ember/component';
+import Component from "@ember/component";
 
 export default Component.extend({
   didRender() {
```

Comment:
This doesn't exist anymore in Glimmer components and should use a modifier instead.